pkg/create/templates/readme.go

```diff
@@ -0,0 +1,91 @@
+package templates
+
+// Readme is a default README laid down by s2i create
+const Readme = `
+# Creating a basic S2I builder image  
+
+## Getting started  
+
+### Files and Directories  
+| File                   | Required? | Description                                                  |
+|------------------------|-----------|--------------------------------------------------------------|
+| Dockerfile             | Yes       | Defines the base builder image                               |
+| s2i/bin/assemble       | Yes       | Script that builds the application                           |
+| s2i/bin/usage          | No        | Script that prints the usage of the builder                  |
+| s2i/bin/run            | Yes       | Script that runs the application                             |
+| s2i/bin/save-artifacts | No        | Script for incremental builds that saves the built artifacts |
+| test/run               | No        | Test script for the builder image                            |
+| test/test-app          | Yes       | Test application source code                                 |
+
+#### Dockerfile
+Create a *Dockerfile* that installs all of the necessary tools and libraries that are needed to build and run our application.  This file will also handle copying the s2i scripts into the created image.
+
+#### S2I scripts
+
+##### assemble
+Create an *assemble* script that will build our application, e.g.:
+- build python modules
+- bundle install ruby gems
+- setup application specific configuration
+
+The script can also specify a way to restore any saved artifacts from the previous image.   
+
+##### run
+Create a *run* script that will start the application. 
+
+##### save-artifacts (optional)
+- Create a *save-artifacts* script which allows a new build to reuse content from a previous version of the application image.
+
+##### usage (optional) 
+Create a *usage* script that will print out instructions on how to use the image.
+
+##### Make the scripts executable 
+Make sure that all of the scripts are executable by running *chmod +x s2i/bin/**
+
+#### Create the builder image
+The following command will create a builder image named nginx-centos7 based on the Dockerfile that was created previously.
+` +
+	"```\n" +
```

Comment:
Would be nice for this text and the equivalent README.md to be as closely in sync as possible (bar the {{.ImageName}}s)